plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -350,15 +377,33 @@ def get_isolation_state(self):
                 port_state.update(Constants.STATE_ISOLATED, Constants.ISSUE_OONOC)
                 self.ports_states[port] = port_state
 
+    def start_telemetry_session(self):
+        self.logger.info("Starting telemetry session")
+        response = self.ufm_client.start_dynamic_session(Constants.PDR_DYNAMIC_NAME, self.telemetry_counters, self.t_isolate)
+        if response and response.status_code == http.HTTPStatus.ACCEPTED:
+            port = str(response.content)
+        else:
+            self.logger.error(f"Failed to start dynamic session: {response}")
+            return False
+        return port       
+
+    def run_telemetry_get_port(self):
+        while not self.ufm_client.running_dynamic_session(Constants.PDR_DYNAMIC_NAME):
+            endpoint_port = self.start_telemetry_session()
+            time.sleep(20)
```

Comment:
timeout (20 seconds) should be configurable - depending on the fabric size...

Reply:
it is not related to the discovery but to a sleep in the dynamic